src/Platform/AcquiaCloudMultiSitePlatform.php

```diff
@@ -50,25 +50,42 @@ public function execute(Command $command, InputInterface $input, OutputInterface
     $sshUrl = $environment->sshUrl;
     [, $url] = explode('@', $sshUrl);
     [$application] = explode('.', $url);
+
     $sites = $this->getPlatformMultiSites($environment, $application, $output, $vendor_path[$env_id]);
     if (!$sites) {
       $output->writeln('<warning>No sites available. Exiting...</warning>');
       return 1;
     }
 
+    $group_name = $input->getOption('group');
     $input_uri = $input->getOption('uri');
-    if ($input_uri) {
+
+    if ($group_name && $input_uri) {
+      $helper = $command->getHelper('question');
+      $question = new ConfirmationQuestion('You have provided both the options, group as well as uri. We will ignore the uri option. Do you want to proceed (y/n)?', TRUE);
+      if (!$helper->ask($input, $output, $question)) {
+        return 2;
+      }
+    }
+
+    if ($group_name) {
+      $sites = $this->filterSitesByGroup($group_name, $sites, $output);
+      if (empty($sites)) {
+        return 3;
+      }
+    }
+    elseif ($input_uri) {
```

Comment:
Do not use `elseif` unless it is unavoidable!! I think we can get rid off it here.